src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtils.java

```diff
@@ -468,4 +468,45 @@ public static String getFileExtension(String fileNameWithExtension) {
         }
         return extension;
     }
+
+    /**
+     * Checks whether the given string is a valid identifier.
+     * @param str A string to check.
+     * @return true when the given string contains valid identifier.
+     */
+    public static boolean isIdentifier(String str) {
+        boolean isIdentifier = false;
+
+        if (!str.isEmpty()) {
+            isIdentifier = true;
```

Comment:
Sure, but we've already discussed this line: https://github.com/checkstyle/checkstyle/pull/3924#discussion_r106959659

Are you sure, that it should be reverted?